### DISCUSSION TITLE
Publii Stability on k/ubuntu

### DISCUSSION THREAD

**Discussion by meika** [GetPublii/Publii#1925] [Help]
I have recently moved to Kubuntu from nixos with KDE, primarily to have a more easily update-able Publii (i am actually thankful for the deb package) (I kept borking the channel update thingo in nixos).

However there is a noticeable stability issue, as in, Publii just goes poof, very regularly, after some period of time, this varies a lot. No hang, no freeze, just poof. It seem to be the only app doing it every time the app is run.

Some tips on what to do when I notice it go poof to capture some data would be grand.

Publii Version: 0.46.3 (build 17008)

Operating System: Kubuntu 24.10
KDE Plasma Version: 6.1.5
KDE Frameworks Version: 6.6.0
Qt Version: 6.6.2
Kernel Version: 6.11.0-13-generic (64-bit)
Graphics Platform: Wayland
Processors: 4 × Intel® Core™ i5-6500T CPU @ 2.50GHz
Memory: 7.6 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 530
Manufacturer: HP
Product Name: HP ProDesk 600 G2 DM

**Comment by bjazmoore** [comment]
Linux is the most expansive of all platforms, and probably the hardest to do good comprehensive testing.  I noticed your system has plenty of resources which is good.  There is a chance that the stability issues may be dependency related - meaning that it is not actually Publii but the tools that Publii depends on that is crashing.  These are the middleware pieces that Publii depends on - 
node.js (20.*)
npm (>= 10.*)
python (>= 2.5.0 && < 3.12.0)
electron (in version used by Publii), electron-packager, node-gyp and gulp node.js modules installed globally.

Sorry - not an answer, just some ideas.  I think the troubleshooting might depend on other well verses Linux users digging deep.

**Comment by meika** [comment]
Thanks for your reply.
 not sure I have any of those middlewares, which might explain a lot, lol,
or nothing (outside of the deb package I guess)

it is slightly more stable in recent days, install is about a week old
(system & Publii).

I installed the Publii .deb packaged using GDebi as the package manager/app
store app called Discover on kubuntu keeps being throttled (I think) by my
ISP, it just canot update unless I USBC cable tether the local G4 data from
my phone, although not even then (just checking as I type, thinks are the
Kubuntu repositories down?

One should be able to install from that Discover but it get completely
distract by the need to refresh repository info, or just check if it has
changed... Can't install anything from it... <wrong forum for this vent>

But I never leave Publii running, & basically save to disk all the time
like it is 1987.

meika

**Comment by dziudek** [comment]
@meika - unfortunately without any logs it will be very hard to find a reason of such behaviour. Most probably the app returns some error, but it is not logged in and it causes the problem. Please start with opening Publii from console - maybe some errors will appear when the app will disappear without interaction.

**Comment by meika** [comment]
Ok, I'll check it out. and wait and watch

thank you

**Comment by meika** [comment]
ooh, had a bad poof, ran it fromthe konsole and the konsole went poof too,

will google how to get thelogs whenI get time (noticed the poof after some
time.... lost quite a bit too, I do my final edit in Publii)

IS there anyone here with experience as to which distro is more
stable running Publii ? I am basically happy to distro jump to support
Publii

**Reply by carstenfeller** [reply]
Hi @meika Publii AppImage from [https://getpublii.com/download/](https://getpublii.com/download/) running on Linux Mint 21.3 without any problems for like some months.
Try to use the Publii.AppImage (do not forget to make it executable) and avoid .rpm or .deb for most scenarios.

**Comment by meika** [comment]
Thanks for the tip!

Will give it a go

meika

**Comment by meika** [comment]
Found a space to take publii out of production...

1. Uninstalled the .deb
2. backup-ed my publii document directory
3. from https://getpublii.com/download/  downloaded publii.appimage, made it executeable,

 nothing,
downloaded again, nothing
restart, still nothing on double clicking the appimage (file manager dolphin appears to slow down but nothing happens) 

looks up fuse,  Kubuntu 24.10 has fuse3
ii  fuse3                                                    3.14.0-9                                    amd64        Filesystem in Userspace (3.x version)
ii  libfuse3-3:amd64                                         3.14.0-9                                    amd64        Filesystem in Userspace (library) (3.x version)

where do I go to find out publii.appimge requirements?

do I need to install a lower fuse along fuse3? as per 
https://docs.appimage.org/user-guide/troubleshooting/fuse.html#how-to-install-fuse

will go back to the .deb package despite instability

**Reply by carstenfeller** [reply]
Just to be clear, you made it executable like desribed here at "how to run an appimage": https://docs.appimage.org/introduction/quickstart.html

Did you try other appimages, f.e. from here: https://keepassxc.org/download/#linux ? How do they behave?

Never heard of this before... Does Kubuntu handle appimage related stuff different from Ubuntu/ Mint?

Try Linux Mint Live from a usb stick, this shoud absolutely work right out of the box...

**Comment by meika** [comment]
Thanks I'll give it a go (again)

Yes I made it executable.  And I can't imagine it is different at all.

Part of this journey was trialing Mint on USB but it just failed,, the
whole installer quit on me during selection of locales etc, & perhaps due
to the ISP related issues. It then refused to boot from the USB stick, so
I'll try a new one.

As time goes by the deb package seems to be more stable, did manage to get
some updates through eventually.
